src/rules.ts

```diff
@@ -268,7 +268,7 @@ class MatchingRules extends Array<MatchingRule> {
 
     const matchedRules = await Promise.all(matchingRules);
 
-    const filtered = matchedRules.filter((rule) => isMatchingRule(rule) && rule.matched);
+    const filtered = matchedRules.filter((rule) => isMatchingRule(rule) && rule.matched || rule.action === RuleActions.status);
```

Comment:
This should be a group to future proof the concept. 



```suggestion
const alwaysMatchedRulesTypes = [RuleActions.status];
const filtered = matchedRules.filter((rule) =>  alwaysMatchedRulesTypes.includes(rule.action) || isMatchingRule(rule) && rule.matched);
```

Reply:
you are right, i did the change